client/package.json

```diff
@@ -24,7 +24,7 @@
     "crypto"
   ],
   "dependencies": {
-    "@axiom-crypto/circuit": "2.0.5",
+    "@axiom-crypto/circuit": "link:../circuit/dist",
```

Comment:
Is this intentional?

Reply:
yes, local links work in this repo now because the publish-all script will sanitize